cli/lib/kontena/cli/nodes/ssh_command.rb

```diff
@@ -3,9 +3,9 @@ class SshCommand < Kontena::Command
     include Kontena::Cli::Common
     include Kontena::Cli::GridOptions
 
-    parameter "NODE_ID", "Node id"
+    parameter "[NODE_ID]", "SSH to Grid node, or --any"
```

Comment:
> Also maybe it should be [NODE_NAME]?

All of the existing `kontena node ...` commands use `NODE_ID`... which is indeed odd, because the "node id" is like `44C7:P5OM:NBJT:WXHV:6EDU:67T5:YDMX:4YPU:PF6D:VUH5:7LE7:`